src/openrct2-ui/windows/Scenery.cpp

```diff
@@ -832,8 +876,16 @@ class SceneryWindow final : public Window
             }
         }
 
+        // Sort scenery group tabs before adding other tabs
+        SortTabs();
+
         // Add misc tab
         _tabEntries.emplace_back();
+        _tabEntries.back().Type = SCENERY_TAB_TYPE_MISC;
+
+        // Add all tab
+        _tabEntries.emplace_back();
+        _tabEntries.back().Type = SCENERY_TAB_TYPE_ALL;
```

Comment:
I think it's neater to emplace them with the correct value already set, and combine these two since it's pretty obvious what they do:
```suggestion
        // Add misc and all tab
        _tabEntries.emplace_back(SceneryWindow::SceneryTabInfo{ SCENERY_TAB_TYPE_MISC });
        _tabEntries.emplace_back(SceneryWindow::SceneryTabInfo{ SCENERY_TAB_TYPE_ALL });
```